arch/sparcv8leon3/jmp.S

```diff
@@ -0,0 +1,49 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * setjmp, longjmp (sparcv8leon3)
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Lukasz Leczkowski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#define __ASSEMBLY__
+
+
+.section ".text"
+.align 4
+
+/* TODO */
+.globl _setjmp
+.type _setjmp, #function
+_setjmp:
+	nop
```

Comment:
These stubs are invalid - all these unimplemented functions are functions - i.e. we need to return from them (and preferably set return value to `-ENOSYS`). Right now after executing `nop` CPU will runaway and execute some random code it finds that's laying after these stubs